desktop/npm-helper.js

```diff
@@ -364,18 +481,20 @@ function updatedFonts () {
       }
     })
 
-  icons['iconfont-proof-new'] = {isFont: true, gridSize: icons['iconfont-proof-good'].gridSize}
-  icons['iconfont-proof-followed'] = {isFont: true, gridSize: icons['iconfont-proof-good'].gridSize}
+  const grids = svgToGridMap()
+  let charCode = 0xe900
 
-  const findCode = /\.icon-kb-(.*):before {\n {2}content: "\\(.*)";/g
-  while (true) {
-    const match = findCode.exec(fs.readFileSync('./renderer/fonticon.css', {encoding: 'utf8'}))
-    if (!match) {
-      break
-    }
-    const [, name, charCode] = match
-    icons[name].charCode = parseInt(charCode, 16)
-  }
+  Object.keys(grids).forEach(gridSize => {
+    Object.keys(grids[gridSize]).forEach(name => {
+      const info = grids[gridSize][name]
+      icons[`iconfont-${info.name}`] = {
+        isFont: true,
+        gridSize: info.gridSize,
+        charCode,
```

Comment:
this isn't true as the charCode mutates across the outter loop so it'd need to be aware of that also which makes the code more complex